components/slack/join-form.js

```diff
@@ -13,6 +13,9 @@ const JoinForm = ({ sx = {} }) => {
       : { reason: router.query.reason }
   })
 
+  const isAdult = useField('educationLevel').value == 'tertiary'
+  const useWaitlist = process.env.NEXT_PUBLIC_OPEN == 'true'
```

Comment:
```suggestion
  const useWaitlist = process.env.NEXT_PUBLIC_OPEN != 'true'
```

I think we need to flip this over. The semantics aren't great but from memory when the Slack isn't "open" we use the waitlist